pd-client/leader_change_test.go

```diff
@@ -111,6 +120,49 @@ func (s *testLeaderChangeSuite) TestLeaderChange(c *C) {
 	c.Error("failed getTS from new leader after 10 seconds")
 }
 
+func (s *testLeaderChangeSuite) TestLeaderTransfer(c *C) {
+	servers, endpoints, closeFunc := s.prepareClusterN(c, 3)
```

Comment:
seem we can use 2 server here. So if we delete the leader, another can become leader directly, then we delete the new leader, the previous old leader may become leader again.